WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -384,9 +382,8 @@ public boolean onPreferenceClick(Preference preference) {
         } else if (preference == mCategoryPref || preference == mFormatPref) {
             return !shouldShowListPreference((DetailListPreference) preference);
         } else if (preference == mExportSitePref) {
-            showExportContentDialog();
         } else if (preference == mDeleteSitePref) {
-            showDeleteSiteDialog();
+            requestPurchasesForDeletionCheck();
```

Comment:
I don't know if there is a separate issue/PR for this, but I think we should show some indication that we're making a request to the server. When the user taps on "Delete site", especially on a slow connection, it looks like nothing is happening.